app/kubemci/pkg/kubeutils/utils.go

```diff
@@ -74,12 +74,16 @@ func getClientsForContexts(kubeconfig string, kubeContexts []string) (map[string
 	return clients, err
 }
 
+// ExecuteCommand executes the command in 'args' and returns the output and error, if any.
 // Extracted out here to allow overriding in tests.
-var executeCommand = func(args []string) (string, error) {
+var ExecuteCommand = func(args []string) (string, error) {
 	glog.V(3).Infof("Running command: %s\n", strings.Join(args, " "))
-	output, err := exec.Command(args[0], args[1:]...).CombinedOutput()
+	// TODO(nikhiljindal): Figure out how to use CombinedOutput here to get error message in output.
+	// We dont use it right now because then "gcloud get project" fails.
+	output, err := exec.Command(args[0], args[1:]...).Output()
 	if err != nil {
-		glog.V(3).Infof("%s", output)
+		err = fmt.Errorf("unexpected error in executing command '%s': error: %s, output: [%s]", strings.Join(args, " "), err, output)
```

Comment:
You know this already, but just reminding: use `%q` instead of `'%s'`?

Reply:
Done